src/app/story/directives/basic.component.ts

```diff
@@ -48,6 +48,17 @@ import { StoryModel, TabService } from '../../shared';
       tiptext="More is better!" >
         <div class="fancy-hint">A comma-separated list of tags relevant to the content of your piece.</div>
       </publish-fancy-field>
+      
+      <publish-fancy-field label="Release Date">
+        <div class="fancy-hint">If the release date is not set manually,
+          the current date will be used for publishing.</div>
+        <publish-datepicker 
```

Comment:
Am I missing some CSS?  When I focus this field, the popup seems to have a transparent background -> http://cav.is/s/5vio7.jpg